lib/schemas/AuthenticationSchema.js

```diff
@@ -25,28 +39,36 @@ module.exports = makeSchema({
         'custom',
         'digest',
         'oauth2',
-        'session'
-      ]
+        'session',
+      ],
     },
     test: {
       description: 'A function or request that confirms the authentication is working.',
       oneOf: [
         {$ref: RequestSchema.id},
-        {$ref: FunctionSchema.id}
+        {$ref: FunctionSchema.id},
       ]
     },
     fields: {
       description: 'Fields you can request from the user before they connect your app to Zapier.',
-      $ref: FieldsSchema.id // TODO: can this be DynamicFieldsSchema?
+      $ref: FieldsSchema.id,
+    },
+    connectionLabel: {
```

Comment:
We're going with `Connection Label` in the Web Builder as well, so I went with the same name here, and yeah, it's a new term, AFAIK.